packages/fxa-shared/package.json

```diff
@@ -40,11 +40,11 @@
     "@types/chai": "^4.2.18",
     "@types/chance": "^1.1.2",
     "@types/generic-pool": "^3.1.9",
+    "@types/hapi__joi": "^15.0.4",
```

Comment:
Huh I guess there is no @types/joi package?

Reply:
The types are in the `joi` package itself.